lib/bolt/cli.rb

```diff
@@ -504,6 +508,12 @@ def generate_types
       0
     end
 
+    def initialize_project
+      path = File.expand_path(options[:object] || Dir.pwd)
+      FileUtils.mkdir_p(path)
+      FileUtils.touch(File.join(path, 'bolt.yaml')) ? 0 : 1
```

Comment:
Ugh I think my update comment got lost: I think we should print something here, which means using the `ok` pattern you had but adding a `outputter.print_message("Successfully create Bolt project directory at #{path}")` before returning.